<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QWidget" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #000;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IRANSansFaNum Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #FFF;</string>
   </property>
   <property name="text">
    <string>يك عدد از 1 تا 20 حدس بزن</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_input">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>80</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IRANSansFaNum Black</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #FFF; border-radius: 10px; border: 0px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_guess">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IRANSansFaNum Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #FFFF00;background-color: #FF1111; border-radius: 5px; border: 0px;</string>
   </property>
   <property name="text">
    <string>حدس</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IRANSansFaNum Black</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #FFFF00;</string>
   </property>
   <property name="text">
    <string>فرصت:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_joon">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IRANSansFaNum Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #FFF;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_comment">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IRANSansFaNum Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px; color: #FFF;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_new">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IRANSansFaNum Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000; background-color: #FFF; border-radius: 5px; border: 0px;</string>
   </property>
   <property name="text">
    <string>بازی جدید</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
